src/views/microworld_art/microworld_art.jsx

```diff
@@ -0,0 +1,4 @@
+var render = require('../../lib/render.jsx');
+var Microworld = require('../../components/microworld/microworld.jsx');
+
+render(<Microworld microworldData={require("./microworld_art.json")} />, document.getElementById('view'));
```

Comment:
I think the common convention for react is to require ahead of time to get the object, and then load it into the render method later on. @rschamp is that correct as far as you know?